src/utilities.cpp

```diff
@@ -88,8 +88,8 @@ void pasteFilesFromClipboard(const Fm::FilePath& destPath, QWidget* parent) {
     if(paths.empty() && data->hasUrls()) {
         // The KDE way
         paths = Fm::pathListFromQUrls(data->urls());
-        QByteArray cut = data->data("x-kde-cut-selection");
-        if(!cut.isEmpty() && cut.at(0) == '1') {
+        QByteArray cut = data->data(QStringLiteral("application/x-kde-cutselection"));
+        if(!cut.isEmpty() && cut.at(0) == QLatin1Char('1')) {
```

Comment:
@luis-pereira Ok, why I don't have mentioned build definitions present in my build files, I will study later. I will update the commit now, to make it ready first. (again force pushed, it's small to see it whole).